<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDevice</class>
 <widget class="QWidget" name="NewDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新建工程文件</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>474</width>
        <height>587</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>基本配置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="5" column="1" colspan="2">
           <widget class="QComboBox" name="comboBoxComMode">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentText">
             <string>网络</string>
            </property>
            <item>
             <property name="text">
              <string>网络</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>串口</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxUpdateCycle">
            <property name="prefix">
             <string/>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>通信方式：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QComboBox" name="comboBoxUpdateCycle">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>毫秒</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>秒</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>分</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditDeviceDescribe"/>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEditDeviceName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>更新周期：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>设备描述：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>设备名称：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>超时时间：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>设备地址：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QComboBox" name="comboBoxTimeout">
            <property name="currentText">
             <string>毫秒</string>
            </property>
            <item>
             <property name="text">
              <string>毫秒</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>秒</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>分</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxTimeOut">
            <property name="value">
             <double>3.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>通讯协议：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QComboBox" name="comboBoxComProtocol">
            <item>
             <property name="text">
              <string>标准MODBUS</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBoxDeviceAddr">
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>高级配置</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QComboBox" name="comboBoxComInterval">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string>毫秒</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>秒</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxComInterval">
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>命令间隔时间：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>数据包采集失败重试次数：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxReconnectNum">
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>次</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>设备连续失败</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>次转为掉线</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spinBoxFailNum">
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>故障恢复周期：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBoxFailRecovery">
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>秒</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxNet">
         <property name="title">
          <string>网络：</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>端口：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPort">
            <property name="inputMask">
             <string>0000</string>
            </property>
            <property name="text">
             <string>502</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditTcpIp">
            <property name="inputMask">
             <string>000.000.000.000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>设备IP:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxSerial">
         <property name="title">
          <string>串口：</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label_17">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>串口：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>波特率：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_6"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>奇偶校验：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_8"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>停止位：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_10"/>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="comboBox_7"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>数据位：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QComboBox" name="comboBox_9"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonOk">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCancle">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditDeviceName</tabstop>
  <tabstop>lineEditDeviceDescribe</tabstop>
  <tabstop>doubleSpinBoxUpdateCycle</tabstop>
  <tabstop>doubleSpinBoxTimeOut</tabstop>
  <tabstop>spinBoxDeviceAddr</tabstop>
  <tabstop>comboBoxComMode</tabstop>
  <tabstop>comboBoxComProtocol</tabstop>
  <tabstop>doubleSpinBoxComInterval</tabstop>
  <tabstop>spinBoxReconnectNum</tabstop>
  <tabstop>spinBoxFailNum</tabstop>
  <tabstop>spinBoxFailRecovery</tabstop>
  <tabstop>lineEditTcpIp</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancle</tabstop>
  <tabstop>comboBox_7</tabstop>
  <tabstop>comboBoxComInterval</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>comboBox_9</tabstop>
  <tabstop>comboBoxTimeout</tabstop>
  <tabstop>comboBoxUpdateCycle</tabstop>
  <tabstop>comboBox_8</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>comboBox_10</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
